packaging/macOS/freeze_program.py

```diff
@@ -9,11 +9,14 @@
 import subprocess
 from hashlib import sha256
 from pathlib import Path
+import logging
 
+logger = logging.getLogger()
```

Comment:
I've used `logger` because I've had hard time to differentiate outputs from script against the outputs of the sub-commands

Reply:
One other problem with not using logger => print is not flushed by default on each line

Reply:
> I've used logger because I've had hard time to differentiate outputs from script against the outputs of the sub-commands

That's why there is those `### foo ###` & `>>> my command` patterns, if we want to go one step further we could add color but I find this simple and readable enough for the time being (and I debugged A LOT this script ^^)

Reply:
So anyway, I've added color

Reply:
Please stop adding unecessary complexity to this PR, just revert those changes and stick with good old uncolored print for the moment ;-)

Reply:
One other problem with not using logger => print is not flushed by default on each line

Reply:
> I can add flush=True to print but I don't like it

Do it, this is a clean&simple solution to the issue (you can also add a `# Use flush to avoid mixing with subprocess output` comment the first time the print is used)

Reply:
Creating an helper function wouldn't be simpler ?

Reply:
> Creating an helper function wouldn't be simpler ?

Yup ;-)